config/modules/HelpDesk.php

```diff
@@ -13,4 +13,6 @@
 	'HIDE_SUMMARY_PRODUCTS_SERVICES' => false,
 	'DEFAULT_VIEW_RECORD' => 'LBL_RECORD_PREVIEW',
 	'CONTACTS_CHECK_EMAIL_OPTOUT' => true,
+	// Hide summary products services bookmark
+	'HIDE_SUMMARY_PRODUCTS_SERVICES' => false,
```

Comment:
?

Reply:
https://github.com/YetiForceCompany/YetiForceCRM/blob/e624ac7a6275ea030eef3e4a019161650dc0a4fd/config/modules/Accounts.php#L16
https://github.com/YetiForceCompany/YetiForceCRM/blob/e624ac7a6275ea030eef3e4a019161650dc0a4fd/config/modules/SSalesProcesses.php#L15

Missing in HeplDesk